code/dynamic_programming/test/longest_repeating_subsequence/longest_repeating_subsequence_test.cpp

```diff
@@ -0,0 +1,17 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#ifndef LONGEST_REPEATING_SUBSEQUENCE_TEST
+#define LONGEST_REPEATING_SUBSEQUENCE_TEST
```

Comment:
Again, I have no idea why you are using _header_ guards, because that isn't what they are for.